.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: go
+services:
+- docker
 go:
-  - 1.9.x
+- 1.9.x
 install:
-  - go get github.com/golang/lint/golint
+- go get github.com/golang/lint/golint
 script:
-  - make clean check
\ No newline at end of file
+- make clean check
+after_sucess:
+- if [ "$TRAVIS_BRANCH" == "master" ]; then
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD" quay.io;
+    make push/kube-aws-approver;
+  fi
+env:
+  matrix:
+  - secure: CpPqniMX6G95JiVKi2fJEeR0W6Cke81ay3tUKc/DDHN0Saag7IVDM954PnzbAAyVu8xb1+A3KHA4xKSA+xUyXrgk3mfxH5JpvGHu56rG8lfJXYZEWDio8IlFNpWczExKtAIxo4GbLZyj67eyLsL8fb1ITvYjr3S3MLwLvbemJN18XGVwyrOY4BGL0qbTeOA1gP+GuvWusArr5cvzcwoy6fFGEJpHeF4HmkH7nQ5EJF0jm3VcdOfjbDKodV9wLx1MILMCZPoQFbPw4vBBLqPsGcS38zEdBIaJ50me/tU8qctY8OqNt4hqrgsungBW0QNVQEcUxcPQU3TUo6UjBqnJIsHtgZRsuBlNs6xTyLvEHkBc0VjOVl6CHKBCAJfXC13RiY6DvE5uWoWF7xz60h7Sq2sVpoYHQ5cxU0uR/DWtjjIALPejY5EOrvKeg+pn+cmYYBFPLaWKCxQWD0fQko7/nYeUJqdZmxjjZmmsQfKC0dQRfg8XZ99ppnaJVRwz7txGIqbdaSlXJ1o2jW7cfxY0BMbvVM3uPeNH+PCPxoFMCa1ck1NTsCuDjgVOuJkEZkNCX2KfrjVCUYr4SRTbF0nQMqh6my3XDmMMCOFQRa6+gndm/1XLMjjJUMlwnSFucUg08GSPoVhBM4NilsRVYMA2NXlwcXe2EFiJrl17QNcoDu4=
```

Comment:
Can you add some comments about what these are?

Reply:
:+1: